public/app/features/alerting/unified/Templates.test.tsx

```diff
@@ -97,8 +105,8 @@ describe('Templates K8s API', () => {
 
     expect(form).toBeInTheDocument();
     expect(within(form).getByRole('textbox', { name: /Template name/ })).toHaveValue('custom-email (copy)');
-    expect(within(form).getAllByTestId('code-editor')[0]).toHaveValue(
-      "{{ define 'custom-email' }}  Custom email template {{ end }}"
+    expect(within(form).getAllByTestId('code-editor')[0]).toHaveTextContent(
+      '{{ define "custom-email_NEW" }} Custom email template {{ end }}'
```

Comment:
this test was wrong on the parent because of the wrong mock. When duplicating a template we parse the define and we replace witt a new generated name (using `updateDefinesWithUniqueValue`)